src/bumerang/borrowhandler.py

```diff
@@ -3,9 +3,11 @@
 
 
 class BorrowHandler(BumerangRequestHandler):
-    """This class is here to handle and requests made by the user in attempt to
+    """This class is here to handle any requests made by the user in attempt to
        borrow an item or something else that may be time sensitive.
     """
+    Borrow = 0
+    Lend = 1
```

Comment:
This doesn't need to be done for this PR, but we can create an enum type to be more semantic. [Source](https://docs.python.org/3/library/enum.html)